Dmitriy_Vysotskyy2.0/Tests/AddToBasketTest.cs

```diff
@@ -0,0 +1,41 @@
+using Dmitriy_Vysotskyy2._0.Models;
+using Dmitriy_Vysotskyy2._0.PageObjects;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace Dmitriy_Vysotskyy2._0.Tests;
+
+public class AddToBasketTest 
```

Comment:
Let's remove all tests inside Tests folder and leave BDD features only. We don't need extra duplication

Reply:
or you can remove tests, but left folder for API tests